Dockerfile

```diff
@@ -1,11 +1,14 @@
 # Build this Docker container from `protocol` directory with:
 #   docker build -t <username>/<imagename> .
 # Execute the voting system with:
-#   docker run <username>/<imagename>
-#     --network=<network>
+#   docker run
 #     --env SENDGRID_API_KEY=<key>
 #     --env NOTIFICATION_FROM_ADDRESS=<email address>
 #     --env NOTIFICATION_TO_ADDRESS=<email address>
+#     --env LOCALHOST=<ethereum host>
```

Comment:
If someone wants to use a public node (like infura), how can they provide their keys on the client side? Do they need to run their own node that's preloaded with their keys?

Reply:
Or maybe the goal of this PR has slightly changed?

Reply:
Ah, yeah. I added `LOCALHOST` and `LOCALPORT` to facilitate testing against an external instance of ganache, since a docker image deployed through GCS doesn't have access to a local instance.

You can pass a mnemonic through environment parameters, so we don't need additional code to facilitate that. I just added a comment to make it clear. Of course this isn't necessarily secure.

Reply:
Ahhhh, that makes sense. Thanks for clarifying.